package.json

```diff
@@ -43,31 +40,29 @@
   },
   "devDependencies": {
     "@biomejs/biome": "^1.5.3",
-    "@types/bun": "^1.0.3",
-    "@types/node": "^20.11.5",
+    "@types/bun": "^1.0.4",
+    "@types/node": "^20.11.8",
     "bumpp": "^9.3.0",
     "bun": "^1.0.25",
     "hast-util-to-html": "^9.0.0",
-    "husky": "^8.0.3",
+    "husky": "^9.0.6",
     "mdast-util-to-hast": "^13.1.0",
     "prettier": "^3.2.4",
     "remark": "^15.0.1",
     "tsup": "^8.0.1",
     "typescript": "^5.3.3",
     "vite": "^5.0.12",
-    "vitest": "^1.2.1",
-    "shikiji": "^0.10.0",
-    "shikiji-transformers": "^0.10.0"
+    "vitest": "^1.2.2",
+    "shiki": "^1.0.0-beta.0",
+    "@shikijs/transformers": "^1.0.0-beta.0"
   },
   "peerDependencies": {
-    "shikiji": "^0.7.0 || ^0.8.0 || ^0.9.0 || ^0.10.0"
+    "shiki": "^1.0.0-beta.0"
```

Comment:
Maybe add `|| ^1.0.0` too? Does this cover `beta.1` etc? Unlikely there'll be affecting breaking changes

Reply:
done